frontend/packages/kubevirt-plugin/src/components/vm-templates/os-icons.scss

```diff
@@ -2,3 +2,8 @@
   width: 2rem;
   color: var(--pf-global--palette--gold-300);
 }
+
+.kv-empty-pin-icon {
+  width: 2rem;
+  color: lightgray;
```

Comment:
can you find the color in pattenfly palettet ?
https://www.patternfly.org/2020.08/documentation/overview/global-css-variables#global-css-variables

Reply:
changed